inventory/group_vars/vagrant

```diff
@@ -27,12 +27,15 @@ nodepool_labels:
 nodepool_providers:
   - name: cicloud
     cloud: cicloud
-    max-servers: 10
+    max-servers: 3
     images:
       - name: ubuntu-xenial
         min-ram: 2048
         diskimage: ubuntu-xenial
         private-key: /var/lib/nodepool/.ssh/id_rsa
+    networks:
+      - name: 'nodepool'
+        public: False
```

Comment:
I dont think we want to encourage devs to boot slaves on our production nodepool environment.  We could instead default to the 'internal' network and the slaves should be reachable via fixed ip from the dev env, assuming it to was booted on the internal network.